repair/incremental.cc

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (C) 2015-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+ */
+
+#include "repair/incremental.hh"
+#include "utils/log.hh"
+
+extern logging::logger rlogger;
+
+namespace repair {
+    std::vector<sstables::shared_sstable> filter_sstables(std::vector<sstables::shared_sstable> sstables, repair::sstables_repair_state repair_state, int64_t sstables_repaired_at) {
```

Comment:
No need to spend an indent here. Namespace-scope functions can be at column 0.

Reply:
Yes. Fixed.